---
title: "Research"
permalink: /research/
author_profile: true
---

My research interests include (but are not limited to): Ecosystem Biology, Population Genomics, Bioinformatics, Molecular Ecology, Biodiversity, Environmental Impacts, Climate Change Response, Conservation Policy, & Marine Sciences

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">

## Gulf of Maine Kelp Forests

My PhD research focuses on the kelp forests that line the coast of Maine. The forests are primarily made of two species - horsetail kelp (*Laminaria digitata*) and sugar kelp (*Saccharina latissima*). These foundation species create habitat, provide shelter, and are a food resource for other Gulf of Maine organisms. Alongside their ecological importance, these species have aquaculture value, as the growing aquaculture industry relies on seed from wild kelp forests to generate their crop. 

However, climate change is threatening the survival of these species (with the Gulf of Maine being one of the fastest warming locations in the world) - and how the species respond can have a profound cascading effect on the rest of the ecosystem that they create. It is of utmost importance that we gather a baseline understanding of this ecosystem in the face of our changing climate and growing industry so that we can create conservation and policy measures that simultaneously allow for the protection of this ecosystem and the growth of an industry beneficial to many waterfront workers in Maine.

![Kelp.](/images/KelpTypes.jpg)

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">

## Ocean Twilight Zone

Prior to my Ph.D., I worked as a research assistant at [Woods Hole Oceanographic Institution](https://www.whoi.edu/) for three years. While there, I worked in two labs - Dr. John Stegeman/Dr. Jed Goldstone's Environmental Toxicology Lab as well as Dr. Annette Govindarajan's Molecular Ecology Lab as part of the [Ocean Twilight Zone](https://twilightzone.whoi.edu/) project.

While working with Jed Goldstone, I worked on a plethora of projects: I studied the effects of environmental toxicants, particularly PCBs, using zebrafish and killifish as model organisms. I Assembled marine animal genomes using Oxford Nanopore Technology. I generated and maintained a multitude of wildtype and CRISPR zebrafish lines. Finally, I analyzed the effects of pharmaceuticals and sewage on oysters.

While working with Dr. Govindarajan I spearheaded the development of protocols to use environmental DNA to identify and build a library of eukaryotic animals inhabiting the mesopelagic zone of the ocean. I participated in multiple research expeditions to collect and process eDNA and zooplankton samples across the Atlantic, and collaborated with engineers to design large scale in situ eDNA sampling instrument that is a part of the AUV Mesobot.

![WHOI.](/images/WHOI.jpg)

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">

## Evolution and Developmental Biology

Prior to working at WHOI, I earned my M.S. in Computational Biology as well as my B.S. in Biological Sciences at [Carnegie Mellon University](https://www.cmu.edu/). I was a member of Veronica Hinman's [Evolution, Development, and Regeneration Lab](https://labs.bio.cmu.edu/hinman/), and successfully defended my thesis in 2016. Published as part of a paper in PNAS, my masters thesis focused utilizing computational methods to analyze raw ChIP-Seq data and identify binding locations of transcription factor Tbrain in the bat star (*P. miniata*) and purple sea urchin (*S. purpuratus*) to determine evolutionary significance of presence of secondary binding site in orthologous genes. 

![CMU.](/images/cmu2016.jpg)

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">


