---
title: "Research"
permalink: /research/
author_profile: true
---

My research interests include (but are not limited to): Ecosystem Biology, Population Genomics, Bioinformatics, Molecular Ecology, Biodiversity, Environmental Impacts, Climate Change Response, Conservation Policy, & Marine Sciences

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">

## Gulf of Maine Kelp Forests


My PhD research focused on Kelp forests - foundational ecosystems that dominate temperate marine ecoregions around the globe. These underwater forests flourish in cold, nutrient-rich waters, where they create complex habitats that enhance local species richness, provide refuge to myriad species, and generate energy for coastal food webs. Additionally, these forests are integral to aquaculture, fishing, and ecotourism industries around the world. However, these ecosystems face a multitude of stressors linked to climate change, including the loss of kelp due to thermal stress and the arrival of new species that are detrimental to kelp survival. In the Gulf of Maine, a region incurring rapid ocean warming in conjunction with acute marine heatwaves, kelp forests defined by sugar kelp (*Saccharina latissima*) and horsetail kelp (*Laminaria digitata*) are in decline. Simultaneously, a kelp aquaculture industry – reliant on yearly kelp tissue harvest – has recently emerged in Maine, growing exponentially in the last decade.     

In my dissertation, I implemented genetic analyses to explore how Maine’s kelp forests are responding to climate change. In the first chapter, focused on lacy bryozoan (*Membranipora membranacea*), an invasive species that colonizes kelp and negatively impacts its growth and survival. Using environmental DNA, I investigated the spatial distribution, seasonality, and drivers of lacy bryozoan occurrence, and projected the occurrence of lacy bryozoan in 2055 under various climate scenarios. My second and third chapters explored the population genomics of Maine’s dominant kelp species, to lay a foundation for assessing changes in kelp genetic diversity in response to ocean warming. In Chapter 3, I characterized the genetic diversity, structure, and connectivity of sugar kelp populations across the outer coastal sites in Maine. In Chapter 4, I examined the genetic diversity and population structure of horsetail kelp and the morphologically identical species *Hedophyllum nigripes* across outer coastal sites in Maine. 

This body of work represents the most comprehensive genetic analyses of Maine’s kelp forests to-date. As the Gulf of Maine continues to experience unprecedented change due to ocean warming, this work provides a critical baseline for gauging how these ecologically vital forests are responding. Furthermore, this work provides critical information for shaping regional conservation strategies and supporting science-based management of Maine’s kelp forests.

![Kelp.](/images/KelpTypes.jpg)

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">

## Ocean Twilight Zone

Prior to my Ph.D., I worked as a research assistant at [Woods Hole Oceanographic Institution](https://www.whoi.edu/) for three years. While there, I worked in two labs - Dr. John Stegeman/Dr. Jed Goldstone's Environmental Toxicology Lab as well as Dr. Annette Govindarajan's Molecular Ecology Lab as part of the [Ocean Twilight Zone](https://twilightzone.whoi.edu/) project.

While working with Jed Goldstone, I worked on a plethora of projects: I studied the effects of environmental toxicants, particularly PCBs, using zebrafish and killifish as model organisms. I Assembled marine animal genomes using Oxford Nanopore Technology. I generated and maintained a multitude of wildtype and CRISPR zebrafish lines. Finally, I analyzed the effects of pharmaceuticals and sewage on oysters.

While working with Dr. Govindarajan I spearheaded the development of protocols to use environmental DNA to identify and build a library of eukaryotic animals inhabiting the mesopelagic zone of the ocean. I participated in multiple research expeditions to collect and process eDNA and zooplankton samples across the Atlantic, and collaborated with engineers to design large scale in situ eDNA sampling instrument that is a part of the AUV Mesobot.

![WHOI.](/images/WHOI.jpg)

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">

## Evolution and Developmental Biology

Prior to working at WHOI, I earned my M.S. in Computational Biology as well as my B.S. in Biological Sciences at [Carnegie Mellon University](https://www.cmu.edu/). I was a member of Veronica Hinman's [Evolution, Development, and Regeneration Lab](https://labs.bio.cmu.edu/hinman/), and successfully defended my thesis in 2016. Published as part of a paper in PNAS, my masters thesis focused utilizing computational methods to analyze raw ChIP-Seq data and identify binding locations of transcription factor Tbrain in the bat star (*P. miniata*) and purple sea urchin (*S. purpuratus*) to determine evolutionary significance of presence of secondary binding site in orthologous genes. 

![CMU.](/images/cmu2016.jpg)

<hr style="height:3px;border-width:0;color:#9fedd7;border-color:#9fedd7;background-color:#9fedd7">


